src/services/CicService.ts

```diff
@@ -306,29 +306,30 @@ export class CicService {
 		}
 	}
 
-	private mapCICNames(givenNames: string[], familyNames: string[]): PersonIdentityName[] {
+	private mapCICNames(givenNames: string[], familyName: string): PersonIdentityName[] {
 		const nameParts: PersonIdentityNamePart[] = [];
+	
+		const validateName = (name: string) => {
+			if (!Constants.GIVEN_NAME_REGEX.exec(name)) {
```

Comment:
.test() would be okay to use here as it would return us a boolean value on regex matching. We can use `exec` if were to use the matched values

Reply:
Okay, I was following the advice of SonarCloud, I've changed it to` .test()` now